serial_protocol.md

```diff
@@ -0,0 +1,72 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw value
+  * If response is ok and a raw value prefixed by count of bytes in response. 
```

Comment:
True, since the `GET` command already have variant result (either an int, or a string, or a boolean or some json) maybe we could just have the result being auto promoted to a bulk string once it pass a fixed size? what do you think.